### DISCUSSION TITLE
Real mode support for Win16 programs for Win1.x/2.x?

### DISCUSSION THREAD

**Discussion by Torinde** [ptitSeb/box64#2186] [Ideas]
Is it feasible to add support for Win16 Win1.x/2.x programs that use CPU real mode - like OTVDM?
- https://bugs.winehq.org/show_bug.cgi?id=26715
- https://github.com/otya128/winevdm/issues/1084

**Comment by ptitSeb** [comment]
it would be possible to add 16bits handling, but that's a huge work, so I'm not sure it will be done one day.

**Comment by Torinde** [comment]
It seems @danoon2 [Boxedwine](https://github.com/danoon2/Boxedwine) goes in the direction of such "box16"?
